### PR TITLE
[#172] Add Story URL field to feedback form

### PR DESCRIPTION
**Pull request by noahko96**
#### Addresses #172 

- Add an optional Story URL field to the feedback form.

Screen Shot 2022-05-19 at 9 50 27 AM

### CODE REVIEW

File: db/migrate/20220519134603_add_url_to_pointless_feedback_messages.pointless_feedback.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This migration comes from pointless_feedback (originally 20220518205500)
+class AddUrlToPointlessFeedbackMessages < ActiveRecord::Migration[5.2]
+  def change
+    add_column :pointless_feedback_messages, :url, :string
+  end
+end
```
Comment (efatsi) [inline db/migrate/20220519134603_add_url_to_pointless_feedback_messages.pointless_feedback.rb:6]: did you run the generator to get this migration here?

**Review by efatsi — APPROVED** [review]

File: db/migrate/20220519134603_add_url_to_pointless_feedback_messages.pointless_feedback.rb (hunk 1 above)
Comment (noahko96) [reply db/migrate/20220519134603_add_url_to_pointless_feedback_messages.pointless_feedback.rb:6]: Yes

Comment (noahko96) [reply db/migrate/20220519134603_add_url_to_pointless_feedback_messages.pointless_feedback.rb:6]: Or maybe no, I ran this: `bundle exec rake pointless_feedback:install:migrations`

Comment (noahko96) [reply db/migrate/20220519134603_add_url_to_pointless_feedback_messages.pointless_feedback.rb:6]: Was that the correct way to do that or should I have done something else?

Comment (efatsi) [reply db/migrate/20220519134603_add_url_to_pointless_feedback_messages.pointless_feedback.rb:6]: yeah, that looks right. I've never seen that work for newer migrations that have been added, but cool that it does.

**Pull request merged**